### DISCUSSION TITLE
Label Printer Question

### DISCUSSION THREAD

**Discussion by mbarus** [inventree/InvenTree#4231] [Q&A] [answered]
I was looking through label printers to use with Inventree and was wondering what label printer and labels do people usually use? I saw the thread on the Brother label printer plug in and was leaning towards the Brother QL 800, but if there's something else that works better I would like to know. And is the continuous label roll preferred, or is there a specific label roll that works best? Any help would be appreciated

**Comment by SergeoLacruz** [comment, marked as answer]
We use Zebra printers. They are loaded with rolls of single labels. We use the version with Ethernet interface. So the printer has an IP address and you can print from every computer. I wrote a plugin for the Zebra that is in my repository. 
Describing the label is a bit complicated in html and css but that's how the system works. Some lines of ZPL code might be easier.
We use 50x30mm. That fits well on reels and most boxes while leaving enough space for a QR and human readable info like
IPN, MPN, description and manufacturer. 
I do not know the brother ones.

**Comment by matmair** [comment]
@mbarus you can use brother Q series, zebra and some cups printers.

I am happy with my brother QL800, the NWB models are also fine. My suggestion would be to look at the used market and pick up any of the brands above. Label printers generally don't break so no real need to buy new.

**Comment by miggland** [comment]
Make sure you get a brother with a network connection - they are available with either Wifi or Ethernet, model-specific

**Comment by simonkuehling** [comment]
I am running a Brother QL-720NW with ethernet connection and continuous 62mm labels rolls, which is working really well. Made customized label templates including a QR code and some human readable information (IPN, Name) for a 62x20mm format that fits everywhere and is still easy to scan...
